commons/servers/QueryParameters.ts

```diff
@@ -0,0 +1,73 @@
+import { DeploymentFilters, EntityType, SortingField, SortingOrder } from 'dcl-catalyst-commons'
+import qs from 'qs'
+
+export function toQueryParamsForGetAllDeployments(
```

Comment:
Instead of manually remove the 's' manually, maybe we could do it programmatically? We are doing something similar here
https://github.com/decentraland/catalyst-client/blob/master/src/utils/Helper.ts#L176

That should make things a lot simpler here